hr_skill/views/hr_employee.xml

```diff
@@ -12,7 +12,7 @@
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="arch" type="xml">
             <notebook position="inside">
-                <page string="Skills">
+                <page string="Skills" name="employee_skills" groups="hr.group_hr_user">
```

Comment:
Why don't you add the permissions directly to the view in order to not load the fields?

Reply:
Because there are some public information outside this page should have visible to a person with a basic permission

Reply:
I know, but your code only adds this part. It is a right approach also.

With my comment, the previous view would work perfectly for users outside the group and will be visible only for this one. The modification of the view will be applied only for the users of the group